tests/integration/test_agentd/test_agent_auth_enrollment.py

```diff
@@ -67,13 +120,40 @@ def configure_authd_server(request):
 
 @pytest.mark.parametrize('test_case', tests, ids=[case['description'] for case in tests])
 def test_agent_auth_enrollment(configure_authd_server, configure_environment, test_case: list):
-    """Test different situations that can occur on the agent-auth program during agent enrollment.
+    '''
+    description:
+        Test different situations that can occur on the `agent-auth` program during agent enrollment.
 
-    Args:
-        configure_authd_server (fixture): Initializes a simulated authd connection.
-        configure_environment (fixture): Configure a custom environment for testing.
-        test_case (list): List of tests to be performed.
-    """
+    wazuh_min_version:
+        4.1
+
+    parameters:
+        - configure_authd_server (fixture):
+            Initializes a simulated authd connection.
+
+        - configure_environment (fixture):
+            Configure a custom environment for testing.
+
+        - test_case (list):
+            List of tests to be performed.
```

Comment:
```
parameters:
        - configure_authd_server:
            type: fixture
            brief: Initializes a simulated authd connection.
        - configure_environment:
            type: fixture
            brief: Configure a custom environment for testing.
        - test_case:
            type: list
            brief: List of tests to be performed.
```

May we have to change the `parameters` description
![image](https://user-images.githubusercontent.com/33318890/131659547-02d077b2-8243-45a9-a646-a31bc47d47d6.png)

Reply:
Fixed in: https://github.com/wazuh/wazuh-qa/commit/ca5880b304a4411012747bc4e755facba9db1367